internal/resources/policy/operations/delete.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/scope"
 )
 
-func ResourcePolicyDelete(ctx context.Context, d *schema.ResourceData, m interface{}, rn string) (diags diag.Diagnostics) {
```

Comment:
Since ctx is unused is there a need to name this variable?

Reply:
The ResourcePolicyDelete is being called via [ResourceOperation](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/adfe80d8b841aeb281132346c0df759036bc127a/internal/resources/policy/operations/helper.go#L205) helper function, hence it is needed.